duracloud/management-console
# Adds support for selectively customizing management console emails.

**Adds support for selectively customizing management console emails.**
* * *

**JIRA Ticket**:  https://jira.duraspace.org/browse/DURACLOUD-1177

* Other Relevant Links (Mailing list discussion, related pull requests, etc.)
https://github.com/duracloud/duracloud-db/pull/7

# What does this Pull Request do?
This PR moves the contents of emails into text file-based templates.  The user can override the templates in the management console under "Notifications" in the root console.

# How should this be tested?
Build duracloud-db from the above PR.

To test the customization,  make sure you first apply the database update (./resources/sql/update-4.1.0.sql).
You may modify the template by navigating to the appropriate template in Notifications area of the root console.  You should be able to open the template, make a change, and save it.  When you perform the action that generates an email (such as requesting a password change or creating a new user) the generated email message should reflect the changes you made to the template.

# Additional Notes:
This functionality should be documented:  Here's a ticket to track that work: 

# Interested parties
Tag (@ mention) interested parties or, if unsure, @duracloud/committers
